src/components/availability-panel/drag-drop-context/index.tsx

```diff
@@ -141,8 +146,16 @@ const DragDropcontext: FC<dragDropProps> = ({
           ) : (
             <div>
               <div className={classNames.searchBoxContainer}>
-                <span />
-                {toogleSearch && <input />}
+                <span
+                  onClick={() => {
+                    setToogleSearchName(!toogleSearchName);
+                  }}
+                  aria-hidden="true"
+                  className={classNames.searchText}
+                >
+                  Search
+                </span>
+                {toogleSearchName && <input placeholder="Search by name" value={searchTerm} onChange={(e) => setSearchTerm(e.target.value)} type="text" id="idleMemberSearch" />}
```

Comment:
Since we are repeating **input** let's make it a separate component

Reply:
Sure @Pavangbhat , I'll create a separate component and re-use.

Reply:
@Pavangbhat  
I've converted to a separate component and pushed the code now.